src/components/top-bar-equipment-seach-dialog/top-bar-equipment-search-dialog.tsx

```diff
@@ -93,21 +99,28 @@ export const TopBarEquipmentSearchDialog: FunctionComponent<
         },
         [
             updateSearchTerm,
-            setIsDialogSearchOpen,
+            closeDialog,
             showVoltageLevelDiagram,
             studyUuid,
             snackWarning,
             currentNode,
         ]
     );
 
+    const SuffixRenderer = useCallback(
```

Comment:
```suggestion
    const suffixRenderer = useCallback(
```
because it's a render function not a component ?

Reply:
fixed